src/formulations/inverse-dynamics-formulation-acc-force.cpp

```diff
@@ -142,6 +142,29 @@ bool InverseDynamicsFormulationAccForce::addMotionTask(TaskMotion & task,
   return true;
 }
 
+bool InverseDynamicsFormulationAccForce::addMomentumTask(TaskAngularMomentum & task,
```

Comment:
It seems to me that the AM task is not different from any other motion task, so I don't see the reason for adding custom methods dedicated to it in `InverseDynamicsFormulationAccForce`. Is there a good reason that I am missing?